h/views/accounts.py

```diff
@@ -142,18 +142,12 @@ def _login_redirect(self):
         return self.request.params.get('next', _login_redirect_url(self.request))
 
     def _login(self, user):
-        # Clear any cached feature flags
-        self.request.feature.clear()
```

Comment:
TBH I'm not even sure this was ever a good idea. For the entirety of the request that actually logs the user in, `request.authenticated_userid` is `None` (I think...), and so I'm not even clear that this worked as intended.